host-interaction/hardware/cpu/query-processor-features.yml

```diff
@@ -0,0 +1,44 @@
+rule:
+  meta:
+    name: query processor features
```

Comment:
thoughts on changing to singular: `query processor feature`? otherwise looks good, thanks!

Reply:
That sounds good to me.  I made the changes in `0b8cc29`